assets/js/googlesitekit/datastore/site/info.test.js

```diff
@@ -0,0 +1,199 @@
+/**
+ * core/site data store: site info tests.
+ *
+ * Site Kit by Google, Copyright 2020 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+/**
+ * External dependencies
+ */
```

Comment:
We can remove this as it isn't used. I think we had added this previously as a hack/workaround for the eslint dependency-group rule. This rule isn't currently active as it's no longer included in the recommended set, but we can handle this manually for now. I'd like to add it back once #1135 and https://github.com/WordPress/gutenberg/pull/20629 are merged though.